app/dashboard/models.py

```diff
@@ -1017,13 +1017,7 @@ def reserved_for_user_handle(self, handle):
                 logger.warning(f'reserved_for_user_handle: Unknown handle: ${handle}')
 
         self.bounty_reserved_for_user = profile
-
-@receiver(post_save, sender=Bounty, dispatch_uid="postsave_bounty")
-def postsave_bounty(sender, instance, created, **kwargs):
-    if created:
```

Comment:
a bounty object being created in the DB != a bounty being created in web3!

check bounty.web3_created_on!